slsim/Pipelines/halo_population.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
```

Comment:
can this file be moved to the Deflector folder? I think it's better suited there as it deals with the halo population